nomad/run_scenario.tpl.hcl

```diff
@@ -77,6 +77,12 @@ job "{{ (ds "vars").scenario_name }}" {
     labels   = ["${var.scenario-name}-${group.key}-${group.value}"]
 
     content {
+      restart {
+        interval         = "30m"
```

Comment:
What does the interval define?

Reply:
https://developer.hashicorp.com/nomad/docs/job-specification/restart

[interval](https://developer.hashicorp.com/nomad/docs/job-specification/restart#interval) (string: <varies>) - Specifies the duration which begins when the first task starts and ensures that only attempts number of restarts happens within it. If more than attempts number of failures happen, behavior is controlled by mode. This is specified using a label suffix like "30s" or "1h". Defaults vary by job type, see below for more information.

Reply:
Thanks, so in this combination it limits the overall duration of a job, right?

Reply:
@jost-s Exactly